packages/application-shell/src/components/locale-switcher/locale-switcher.tsx

```diff
@@ -1,15 +1,18 @@
-import { useCallback } from 'react';
+import { useCallback, useState } from 'react';
 import { css } from '@emotion/react';
 import { FormattedMessage } from 'react-intl';
 import type {
   SingleValueProps,
   ValueContainerProps,
   MenuListProps,
 } from 'react-select';
+import GroupHeadingPropsDefinedProps from 'react-select';
```

Comment:
Not sure what props to import here from react-select. At first I imported type `GroupHeadingProps`  but then when running `pnpm build` I'd get this message. 

`🎁 error packages/application-shell/src/components/locale-switcher/locale-switcher.tsx:66:14 - error TS4023: 
Exported variable 'CustomGroupHeading' has or is using name 'GroupHeadingPropsDefinedProps' from external module "merchant-center-application-kit/node_modules/.pnpm/react-select@5.8.0_@types+react@17.0.56_react-dom@17.0.2_react@17.0.2/node_modules/react-select/dist/declarations/src/components/Group" but cannot be named.`

So then I used `GroupHeadingPropsDefinedProps` instead but I still get errors when tests run. So not sure what props I need here. 🤔

Reply:
This default import imports the entire react-select library.

The `GroupHeadingProps` as named import should be correct. If there is an issue, we can figure it out.

Reply:
Okay, thank you, done here -[9866ba3](https://github.com/commercetools/merchant-center-application-kit/pull/3403/commits/9866ba3f23e6e46f1c3c718ac274ebe413735edc) . 
And here's the lint failing - [test](https://github.com/commercetools/merchant-center-application-kit/actions/runs/7843395664/job/21403626500?pr=3403#step:3:219).